pages/common/bup.md

```diff
@@ -0,0 +1,23 @@
+# bup
+
+> Very efficient backup system based on the git packfile format.
+
+- Initialize a backup repository in the specified local directory:
+
+`bup -d {{path/to/repository}} init`
+
+- Index folder "my_folder" before of backup:
```

Comment:
I'm not sure what this does.

Reply:
Index prepare the dir for the backup. Think it as git add, you need do before backup.
```suggestion
- Prepare a folder to the backup:
```

Reply:
Ahhh, ok :-)